plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/explorer/OpenShiftExplorerView.java

```diff
@@ -123,7 +123,8 @@ private Control createExplanationPane(Control connectionsPane, PageBook pageBook
 	private SelectionAdapter onExplanationClicked(final Control connectionsPane, final Control explanationPane) {
 		return new SelectionAdapter() {
 			public void widgetSelected(SelectionEvent e) {
-				ConnectionWizard wizard = new ConnectionWizard();
+				ConnectionWizard wizard = new ConnectionWizard();				
+				wizard.setWindowTitle("New OpenShift Connection");
```

Comment:
@mlabuda ok, then I would still have the call in the contructor, but then have the caller pass the title in as a parameter to the constructor